usefulUtilities/userLocation/userLocation.js

```diff
@@ -0,0 +1,208 @@
+//
+//  userLocation.js
+//
+//  Created by Zach Fox on 2019-04-02
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  See accompanying README.md for usage instructions.
+// 
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+
+(function () {
+    var request = Script.require("./modules/request.js").request;
+
+    var _this;
+    var REFRESH_LOCATION_INTERVAL_MS = 30000;
+    var USER_API_BASE_URL = Account.metaverseServerURL + "/api/v1/users?filter=connections&search=";
+    var TEXT_LOCAL_ENTITY_PROPS = {
+        "type": "Text",
+        // eslint-disable-next-line no-magic-numbers
+        "localPosition": [0, 0, 0.05],
+        // eslint-disable-next-line no-magic-numbers
+        "localRotation": Quat.fromVec3Degrees([0, 0, 0]),
+        "lineHeight": 0.08,
+        // eslint-disable-next-line no-magic-numbers
+        "dimensions": [1.5, 0.2, 0.05],
+        "topMargin": 0,
+        "rightMargin": 0,
+        "bottomMargin": 0,
+        "leftMargin": 0,
+        "grab": {
+            "grabbable": false,
+            "equippableLeftRotation": {
+                "x": -0.0000152587890625,
+                "y": -0.0000152587890625,
+                "z": -0.0000152587890625,
+                "w": 1
+            },
+            "equippableRightRotation": {
+                "x": -0.0000152587890625,
+                "y": -0.0000152587890625,
+                "z": -0.0000152587890625,
+                "w": 1
+            }
+        },
+        "canCastShadow": false,
+        "script": Script.resolvePath("userLocationTextLocalEntity.js?6")
+    };
+
+    var UserLocationBanner = function() {
+        _this = this;
+    };
+
+    UserLocationBanner.prototype = {
+        preload: function(entityID) {
+            _this.entityID = entityID;
+            _this.targetUsername = false;
+            _this.targetDisplayName = false;
+            _this.refreshLocationInterval = false;
+            _this.textLocalEntity = false;
+            _this.canTeleportToTargetUser = false;
+    
+            var properties = Entities.getEntityProperties(entityID, ["userData"]);
```

Comment:
If the only property you are getting is userdata, then maybe just save yourself a var

Reply:
this feels slightly cleaner to me personally!

Reply:
but good point :)